websockets/tcp-chat/src/main.rs

```diff
@@ -228,26 +13,7 @@ async fn main() -> std::io::Result<()> {
 
     // Start tcp server in separate thread
     let srv = server.clone();
-    session::tcp_server("127.0.0.1:12345", srv);
-
-    println!("Started http server: 127.0.0.1:8080");
-
-    // Create Http server with websocket support
-    HttpServer::new(move || {
-        App::new()
-            .data(server.clone())
-            // redirect to websocket.html
-            .service(web::resource("/").route(web::get().to(|| {
-                HttpResponse::Found()
-                    .header("LOCATION", "/static/websocket.html")
-                    .finish()
-            })))
-            // websocket
-            .service(web::resource("/ws/").to(chat_route))
-            // static resources
-            .service(fs::Files::new("/static/", "static/"))
```

Comment:
Since we don't serve `/static` now, we can remove it.

Reply:
thanks for the review! i forgot that part I'll fix it as soon as I get home